openfl/component/ca/downloader.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (C) 2020-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import platform
+import urllib.request
+import shutil
+
+from click import confirm
+
+VERSION = '0.16.0'
```

Comment:
Is there a specific reason to use this version when latest versions are available?

Reply:
The latest `step-ca` release has a different folder structure from what we expect to have in our PKI interface. The transition could not be done smoothly and requires changes in our code that should be addressed in different PR.